cmd/fpga_admissionwebhook/fpga_admissionwebhook.go

```diff
@@ -260,29 +249,29 @@ func main() {
 		config, err = clientcmd.BuildConfigFromFlags(master, kubeconfig)
 	}
 	if err != nil {
-		fmt.Println("Failed to get cluster config ", err)
+		klog.Fatal("Failed to get cluster config ", err)
 		os.Exit(1)
 	}
 
 	patcherManager, err := newPatcherManager(mode)
 	if err != nil {
-		fatal(err)
+		klog.Fatal(err)
```

Comment:
`err` is wrapped here. Should be `klog.Fatalf("%+v", err)` if there's such function.

Reply:
done